src/frontend/komponenter/Felleskomponenter/HeaderMedSøk/FagsakDeltagerskort.tsx

```diff
@@ -45,8 +45,10 @@ const FagsakDeltagerskort: React.FunctionComponent<IFagsakDeltagerskortProps> =
         "BARN_UKJENT": "BARN"
     }
 
+    const kortUtenFagsakId = "gray-out";
+
     return (
-        <div className={`${fagsakId ? "" : "gray-out"}`}>
+        <div className={`${fagsakId ? "" : kortUtenFagsakId}`}>
```

Comment:
```suggestion
        <div className={`${ingenTreff() ? "" : "gray-out"}`}>
```

What about something like this? It is not obvious that !fagsakId means no hit.